src/components/widgets/filesystem/FileSystem.vue

```diff
@@ -393,7 +393,9 @@ export default class FileSystem extends Mixins(StateMixin, FilesMixin, ServicesM
               return false
             }
             break
-
+          case 'moonraker_backup_files':
+            if (file.type === 'file' && file.filename.match(/^.moonraker.conf.bkp$/)) { return false }
```

Comment:
You are using a regular expression here, so "." has to be escaped like "\." as is on the klipper backup files expression just below.

Nevertheless, we do not need a regular expression here, so we can just compare the strings.

```suggestion
            if (file.type === 'file' && file.filename === 'moonraker.conf.bkp') { return false }
```

Reply:
the backup file stats with a "." so cleaned up your update.

Reply:
the file starts with `.` so the hidden file filter will catch this, but I like to see hidden files but hide backup of moonraker, so I like this PR, but will leave it upto you.

Reply:
Good one, I missed the starting "."